ar4_mujoco_sim/mjcf/robot/mj_ar4.mjcf

```diff
@@ -22,110 +22,136 @@
+                      <inertial pos="-0.005 -0.015 0.0" quat="1.0 0.0 0.0 0.0" mass="0.01" diaginertia="5.208333333333334e-07 3.741666666666667e-07 2.2833333333333335e-07" />
+                      <geom name="finger_link_1_collision" pos="0.0 0.0 0.0" quat="1.0 0.0 0.0 0.0" type="mesh" mesh="gripper_finger_collision.stl" class="collision" />
+                      <geom name="finger_link_1_visual" pos="0.0 0.0 0.0" quat="1.0 0.0 0.0 0.0" material="black" type="mesh" mesh="gripper_finger.stl" class="visual" />
+                    </body>
+                    <body name="finger_link_2" pos="0.0 -0.036 0" quat="6.123233995736766e-17 0.0 -1.0 0.0">
+                      <joint name="finger_joint_2" type="slide" class="motor" range="-0.014 0" axis="1 0 0" />
+                      <inertial pos="-0.005 -0.015 0.0" quat="1.0 0.0 0.0 0.0" mass="0.01" diaginertia="5.208333333333334e-07 3.741666666666667e-07 2.2833333333333335e-07" />
+                      <geom name="finger_link_2_collision" pos="0.0 0.0 0.0" quat="1.0 0.0 0.0 0.0" type="mesh" mesh="gripper_finger_collision.stl" class="collision" />
+                      <geom name="finger_link_2_visual" pos="0.0 0.0 0.0" quat="1.0 0.0 0.0 0.0" material="black" type="mesh" mesh="gripper_finger.stl" class="visual" />
+                    </body>
+                  </body>
                 </body>
               </body>
             </body>
           </body>
         </body>
       </body>
+      <body name="dummy" pos="0 0 0" quat="1.0 0.0 0.0 0.0" />
       <site name="base_link_site" pos="0 0 0" quat="1 0 0 0" />
       <camera name="front_camera" mode="track" fovy="90.0" quat="4.329780281177467e-17 4.329780281177466e-17 0.7071067811865475 0.7071067811865476" pos="0.0 2.0 0.5" />
       <camera name="side_camera" mode="track" fovy="90.0" quat="-0.5 -0.4999999999999999 0.5 0.5000000000000001" pos="-2.0 0.0 0.5" />
     </body>
   </worldbody>
 
+  <equality>
+    <joint joint1="finger_joint_1" joint2="finger_joint_2" solimp="0.95 0.99 0.001" solref="0.005 1"/>
+  </equality>
+
   <actuator>
     <motor name="joint_1_ctrl" joint="joint_1" class="motor" />
     <motor name="joint_2_ctrl" joint="joint_2" class="motor" />
     <motor name="joint_3_ctrl" joint="joint_3" class="motor" />
     <motor name="joint_4_ctrl" joint="joint_4" class="motor" />
     <motor name="joint_5_ctrl" joint="joint_5" class="motor" />
     <motor name="joint_6_ctrl" joint="joint_6" class="motor" />
+    <motor name="finger_joint_1_ctrl" joint="finger_joint_1" class="motor" />
+    <motor name="finger_joint_2_ctrl" joint="finger_joint_2" class="motor" />
   </actuator>
 
   <contact>
     <exclude body1="base_link" body2="link_1" />
     <exclude body1="link_1" body2="link_2" />
     <exclude body1="link_2" body2="link_3" />
-    <exclude body1="link_2" body2="link_4" />
     <exclude body1="link_3" body2="link_4" />
     <exclude body1="link_4" body2="link_5" />
-    <exclude body1="link_4" body2="link_6" />
```

Comment:
We still have  collision between these two links, so we need this exclusion.

![Screenshot from 2025-06-18 11-29-42](https://github.com/user-attachments/assets/bdd3d552-7458-4248-ad29-bb9c9ea0fb87)